dizoo/classic_control/pendulum/envs/pendulum_model_env.py

```diff
@@ -0,0 +1,85 @@
+from typing import Any, Union, List, Callable, Dict
+import copy
+import torch
+import torch.nn as nn
+import numpy as np
+
+from ding.envs import BaseEnv, BaseEnvTimestep, BaseEnvInfo, update_shape
+from ding.envs.common.env_element import EnvElement, EnvElementInfo
+from ding.envs.common.common_function import affine_transform
+from ding.torch_utils import to_tensor, to_ndarray, to_list
+from ding.utils import ENV_REGISTRY
+from ding.worker.collector.base_serial_collector import to_tensor_transitions
+
+
+@ENV_REGISTRY.register('pendulum_model')
+class PendulumModelEnv(object):
+
+    def __init__(self, env_id: str, set_rollout_length: Callable, rollout_batch_size: int = 100000):
+        self.env_id = env_id
+        self.rollout_batch_size = rollout_batch_size
+        self._set_rollout_length = set_rollout_length
+
+    def termination_fn(self, next_obs: torch.Tensor) -> torch.Tensor:
+        # This function determines whether each state is a terminated state
+        done = torch.zeros_like(next_obs.sum(-1)).bool()
```

Comment:
why designs the termination condition of pendulum like this?

Reply:
Because there is no terminal state in pendulum. The task is terminated when `max_episode_steps` is reached.
https://github.com/openai/gym/blob/master/gym/envs/classic_control/pendulum.py#L121

This is also true for halfcheetah.
https://github.com/openai/gym/blob/master/gym/envs/mujoco/half_cheetah.py
https://github.com/opendilab/DI-engine/blob/main/dizoo/mujoco/envs/mujoco_model_env.py#L59

Reply:
add some comments about this